### PR TITLE
Initial desktop UI commit

### PR DESCRIPTION
**Pull request by borspn**

### CODE REVIEW

File: Front/RozenaWeather front/Regestration_form.pro
Comment (NazarMarchuk) [inline Front/RozenaWeather front/Regestration_form.pro:1]: delete extra folder

File: Front/RozenaWeather front/web.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    : QWidget{parent}{
+    window = new QWidget();
+    window->setWindowTitle("Вартість біткоїна");
+
+    label = new QLabel(window);
+    label->setGeometry(50, 50, 200, 50);
+
+    manager = new QNetworkAccessManager(this);
+
+    const QUrl url("https://api.coindesk.com/v1/bpi/currentprice/BTC.json");
```
Comment (NazarMarchuk) [inline Front/RozenaWeather front/web.cpp:13]: Contact Rufik for permission to use cryptocurrecy services

**Review by NazarMarchuk — CHANGES_REQUESTED** [review]

File: Front/RozenaWeather front/web.cpp (hunk 1 above)
Comment (borspn) [reply Front/RozenaWeather front/web.cpp]: @NazarMarchuk, I`ve received permission from @rufik_crypto for such use, please reconsider your comment.

**Review by NazarMarchuk — APPROVED** [review]

**Comment by NazarMarchuk** [comment]
For java devs: Package configuration may be erased. 
Reselect Back/src/main/java as a Sources Root

**Pull request merged**